docs/conf.py

```diff
@@ -57,8 +58,7 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-import iris_grib
```

Comment:
The actual Flake8 complaint was about where the `import` was positioned, but I figured I should just replicate what Iris does.